lib/Autocomplete.js

```diff
@@ -322,6 +322,7 @@ let Autocomplete = React.createClass({
           {...this.props.inputProps}
```

Comment:
As I said in the description: 

>If this proves to be problematic we can change the ordering so that the consumer can override it via inputProps: { autoComplete: 'on' }.

Some props we don't want to let the user override (e.g. `onFocus`, `onBlur`), some we *probably* don't want them to override (e.g. `role`, `aria-autocomplete`), while others we *may* want to let them override (e.g. `autoComplete`). I don't know if there has been put any thought into this, so I just followed the existing convention of putting "our" props last.